posydon/utils/ignorereason.py

```diff
@@ -0,0 +1,48 @@
+"""Class for handling ignore reasons when creating new grids."""
+
+
+IGNORE_REASONS_PRIORITY = [
+    'corrupted_binary_history',
+    'corrupted_history1',
+    'corrupted_history2',
+    'ignored_no_BH',
+    'ignore_no_H1',
+    'ignored_scrubbed',
+    'ignore_no_FP',
+    'ignored_no_RLO']
+
+
+class IgnoreReason():
+    """Class for handling ignore reasons when creating new grids."""
+
+    def __init__(self):
+        self.reason = None
+        self.order = None
```

Comment:
I'd prefer not to change this:
1. First of all, infinity could be set to ensure something goes to the edge of the list, so I wouldn't encourage using it instead of `None`
2. You say to use only integers, but `np.inf` is not an integer.
3. The `1 < None` for me is not bad - because it will cause an error and the developer will really start to understand the logic of this object.
4. For the `non-developer`, this is not a problem because they cannot actually access the `.order` property outside the object. So we're safe.